### PR TITLE
fix README.md doc

### PR DESCRIPTION
**Pull request by lengrongfu**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes # https://github.com/Project-HAMi/HAMi/issues/330

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,8 +86,8 @@ HAMi consists of several components, including a unified mutatingwebhook, a unif
 The list of prerequisites for running the NVIDIA device plugin is described below:
 
 - NVIDIA drivers >= 440
-- CUDA Version > 10.2
```
Comment (wawa0210) [inline README.md:89]: Why delete cuda requirement？

Comment (lengrongfu) [reply README.md:89]: image

Comment (wawa0210) [reply README.md:89]: /cc @archlitchi  PTAL

Comment (wawa0210) [reply README.md:89]: /cc @archlitchi

**Pull request merged**